lighthouse-core/audits/manual/pwa-page-transitions.js

```diff
@@ -6,6 +6,17 @@
 'use strict';
 
 const ManualAudit = require('./manual-audit.js');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that suggests a page's transitions should not block network requests. */
```

Comment:
switch to "web page" and positive title format and should give an idea of what transition means? (and they can't not block network requests...the point is to make them *feel* like they don't :)